src/driver/aurora-mysql/AuroraMysqlConnectionOptions.ts

```diff
@@ -37,4 +37,6 @@ export interface AuroraMysqlConnectionOptions
      * (Default: true)
      */
     readonly legacySpatialSupport?: boolean
+
+    readonly poolSize?: never
```

Comment:
why it's `never`?

Reply:
Some drivers doesn't support `poolSize` option. Since I added this option to the `BaseDataSourceOptions` I had to add them as `never`.